build/azure-pipelines/exploration-build.yml

```diff
@@ -1,13 +1,14 @@
 pool:
   vmImage: "Ubuntu-16.04"
 
-trigger: none
+trigger:
```

Comment:
Is this `trigger` change intentional?

Reply:
yup, do you prefer it to be disabled and perform manual merge ?

Reply:
I guess my question is: why do we want to enable merging again for exploration if we merge the E12 code to insiders and thus don't really need exploration anymore?

Reply:
I don't plan on merging this PR, exploration builds will continue from this branch. Plan is to have both exploration and insiders based on 12 for the time being.

Reply:
Ah ok.